test/lib.command.winRate.js

```diff
@@ -440,22 +447,24 @@ test('command.winRate.winRate', t => {
 	});
 
 	t.test('follow up call, one additional battle', st => {
-		const accountInfo = accountInfoMock(100994565, 9260, 13933);
+		const accountInfo = accountInfoMock(100994565, 9260, 13933, 13933000);
 
 		callWinRate(mocks.createMessage(null, 'biggirl22 [CL]'), {
 			account_id: 100994565,
 			wins: 9259,
-			battles: 13932
+			battles: 13932,
+			damage: 13931164
 		}).then(result => {
 			st.deepEqual(result, {
 				sentMsg: [
-					'@biggirl22 [CL], You have won 9260 of 13933 battles. That is 66.46% victory!',
-					'Last time you asked was 1 battles ago, at 66.46% victory.',
-					'Over those 1 battles, you won 100.00%!'
+					'@biggirl22 [CL], You have won 9260 of 13933 battles. That is 66.46% victory! Your average damage is 1000.00.',
+					'Last time you asked was 1 battles ago, at 66.46% victory and 999.94 average damage dealt.',
+					'Over those 1 battles, you won 100.00% with average damage of 1836.00!'
```

Comment:
Believe or not, users commonly run this command after only one battle. Please find a solution that is a bit more intelligent about decimal places.

For example
* A single battle should have a whole number.
* No trailing zeros beyond the natural result. So "1600.4" is written as `1600.4`, but "1600.404" is written as `1600.40`.
* Ignore floating point errors. So "1600.000000000001" is just `1600`.